console/src/main/java/nl/nn/adapterframework/management/web/ServletDispatcher.java

```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2016-2022 WeAreFrank!
+   Copyright 2016-2023 WeAreFrank!
```

Comment:
Dit moet ook nog even worden aangepast voor andere bestanden in dit PR, geloof ik.